homeworks/08-functional-reactive-programming/src/assets.ts

```diff
@@ -0,0 +1,73 @@
+const AnimalSvg = `
```

Comment:
The SVGs used are unoptimised, I suggest using online tools like https://www.svgviewer.dev/ (Optimise -> Pretify).
Inlining SVG as a template string is quite hard to edit the SVG.
![Screenshot 2021-05-31 at 10 42 08](https://user-images.githubusercontent.com/364880/120158498-df777780-c1fc-11eb-8169-3c8a9f892bae.png)

I suggest creating SVG files and importing them as image path and setting `Image` src.


The image loading:
```typescript
import BricksSvg from "./assets/bricks.svg"
import CatSvg from "./assets/cat.svg"
import WindowSvg from "./assets/window.svg"
import ReticleSvg from "./assets/reticle.svg"

function loadSvg(imagePath: string): HTMLImageElement {
    const result = new Image()
    result.src = imagePath
    return result
}

export const BricksImage = loadSvg(BricksSvg)
export const CatImage = loadSvg(CatSvg)
export const WindowImage = loadSvg(WindowSvg)
export const ReticleImage = loadSvg(ReticleSvg)
```

Reply:
Good points. Actually, I took the SVGs from the sample app.